pkg/rancher-desktop/config/settings.ts

```diff
@@ -448,9 +518,23 @@ const updateTable: Record<number, (settings: any) => void> = {
     delete settings.updater;
   },
   5: (settings) => {
```

Comment:
I don't think it matters, since the only people who might be caught midstream between the version 5 we shipped and the current changes are building from source or using CI builds.